cohort-engine/src/test/resources/cql/valueset/FHIRHelpers-4.0.0.cql

```diff
@@ -0,0 +1,269 @@
+library FHIRHelpers version '4.0.0'
```

Comment:
General question on FHIRHelpers - any thoughts on trying to not have a bunch of copies of this in the test folder?  Not sure if there's a reason to lean one way or the other.

Reply:
I started down that path, but stopped because the filesystem resource loader currently supports only a single folder as input. If you need multiple files, as is the case when FHIRHelpers is involved, they have to be in the same folder structure. We could probably improve that, but I decided not to try to solve that in this PR.

Reply:
Thanks for the explanation 👍